src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb

```diff
@@ -3359,19 +3355,7 @@ NextControl:
         ''' <returns></returns>
         ''' <remarks></remarks>
         Protected Function GetProjectPath() As String
-            ' csproj.dll and msbprj.dll implement this Property so first try getting the full path
-            ' through the ProjectProperties. The new CPS based project system doesn't implement this 
-            ' interface. The full path is part of the properties on the BrowseObject and so get it from there.
-            Dim fullPath As String
-            If ProjectProperties IsNot Nothing Then
-                fullPath = ProjectProperties.FullPath
-            Else
-                Dim obj As Object = TryGetNonCommonPropertyValue(GetPropertyDescriptor("FullPath"))
-                fullPath = CType(obj, String)
-            End If
-
-            ' Append a directory separator char as some callsites in Microsoft.VisualStudio.Editors.ClickOnce assume project path ends with it.
-            Return Common.AppendBackslash(fullPath)
```

Comment:
Can you expand on this? Was this a temporary change previously? Was this a hack?